src/main/java/mate.academy/Main.java

```diff
@@ -8,9 +8,8 @@
 public class Main {
 
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductService) injector.getInstance(ProductService.class);
         List<Product> products = productService.getAllFromFile("products.txt");
```

Comment:
The filename 'products.txt' is hardcoded. Consider defining it as a constant or allowing it to be passed as an argument to maintain flexibility and reusability of the Main class.